.github/stale.yml

```diff
@@ -0,0 +1,15 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 365
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 90
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - "Type: Feature"
+# Label to use when marking an issue as stale
+staleLabel: stale
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as "stale" because it has not had
+  any activity for a while. It will be closed in 90 days if no further activity occurs.
+# Comment to post when closing a stale issue. Set to `false` to disable
+closeComment: false
```

Comment:
I think we'd want to enable this, and add to the end the "Thank you  for your contributions." bit from the default stale-bot message.

Reply:
Interesting! If comments are closed, is the original submitter still allowed to open the issue again?

Reply:
I added these changes, but will leave this comment open for the other reviewers to look into...

Reply:
Whoops I misunderstoot and misready. Will deal with this now!